volunteer/views.py

```diff
@@ -155,12 +155,14 @@ def render_username(self, value, record):
         return html_content
 
     def render_teams(self, value, record):
-        """Render the teams as badges."""
+        """Render the teams as badges with links to team detail pages."""
         html_content = ""
         for team in record.teams.all():
+            team_url = reverse("team_detail", kwargs={"pk": team.pk})
             html_content = format_html(
-                '{}<span class="badge bg-secondary">{}</span> ',
+                '{}<a href="{}" class="badge bg-secondary text-decoration-none">{}</a> ',
```

Comment:
I think it's ok to remove the text-decoration-none. The underline helps the user know that it is a link and clickable. Just to make it obvious that the link is clickable.
```suggestion
                '{}<a href="{}" class="badge bg-secondary">{}</a> ',
```